starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,58 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
+<header class="header" id="site-header">
+  
+  <h1>Apple Pie</h1>
+
+</header>
+
+<section class="container section" >
+<article class="description">
+<h2>This was my grandmother's apple pie recipe. I have never seen another one
 quite like it. It will always be my favorite and has won me several first place
 prizes in local competitions. I hope it becomes one of your favorites as well!
+</h2>
+<img src="images/recipe-info.png" alt="recipe-info" class="smallImages">
+<hr>
+</article>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
-
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
+<h2><b>Ingredients</b></h2>
```

Comment:
You can create a div with all the content about the ingredients... same for directions